speech/voices/__init__.py

```diff
@@ -59,3 +59,20 @@ def generate_data_uri(data: bytes, mime_type: str = BINARY_MIMETYPE) -> str:
     """Generate Data URI (RFC2397) from bytes."""
     b64str = b64encode(data).decode("ascii")
     return f"data:{mime_type};base64,{b64str}"
+
+
+# Map locales to a default voice ID
+LOCALE_TO_VOICE_ID = {
+    "is_IS": "Gudrun",
+    "en_US": "Jenny",
+    "en_GB": "Abbi",
+    "de_DE": "Amala",
+}
+assert "is_IS" in LOCALE_TO_VOICE_ID
+
+
+def voice_for_locale(locale: str) -> str:
+    """Returns default voice ID for the given locale. If locale is
+    not supported, returns the default voice ID for 'is_IS'."""
+    vid = LOCALE_TO_VOICE_ID.get(locale)
```

Comment:
Simpler: `return LOCALE_TO_VOICE_ID.get(locale, "is_IS")`

Reply:
This would return "is_IS", whereas the function is supposed to return a voice ID.